src/main/java/cz/muni/pa165/teamwhite/formula1/entity/Car.java

```diff
@@ -71,6 +71,8 @@ public void setDriver(Driver driver) {
 
     public void addComponent(Component component) {
         this.components.add(component);
+
+        component.setCar(this);
```

Comment:
budem to tady teda resit? ... potom by se melo vyresit i mazani v removeComponent, at to odnastavi, pokud to ma byt nejak konzistentni

Reply:
smazu

Reply:
a vyresime komplexne pozdeji